.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build
+env:
+  FILENAME: no-monsters-minecraft_build_${{ github.run_id }}.zip
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+    - main
+    - dev
+  pull_request:
+    branches:
+    - main
+    - dev
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    name: Build
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: Generate a zip package of the datapack
+        run: |
+          zip -r ${{ env.FILENAME }} data img pack.mcmeta README.md INSTALL.md
+          echo "Build status: ${{ job.status }}"
+          
+      - name: Upload the build artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: no-monsters-minecraft_build
+          path: ./${{ env.FILENAME }}
+
+      - name: Download the build artifact
+        uses: actions/download-artifact@v2
```

Comment:
I think the steps after `actions/upload-artifact` are not needed; that was just part of the example you used.

Reply:
Yeah, I'm still getting used to GitHub workflows